pkgs/development/python-modules/adb-enhanced/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, fetchFromGitHub, buildPythonPackage, docopt, psutil, pythonOlder }:
+
+buildPythonPackage rec {
+
+  pname = "adb-enhanced";
```

```suggestion
buildPythonPackage rec {
  pname = "adb-enhanced";
```

@fabaff because GitHub and whitespace 🤷🏼

---

Just saw and added it manually :)